package.json

```diff
@@ -54,7 +54,7 @@
     "@typescript-eslint/parser": "^8.18.0",
     "@vitejs/plugin-react-swc": "^3.1.0",
     "babel-loader": "^8.3.0",
-    "classnames": "^2.3.2",
+    "classnames": "^2.5.1",
```

Comment:
the build output for classnames has a window reference, preventing us from removing `"use client"` directive on components like `Button`. Bumped to see if there was a fix but there wasn't. I think the issue has to do with how vite is building it. Maybe we need to mark it as external but I'd rather explore on a separate branch.